contents/staticAnalysis/ESLint.md

```diff
@@ -0,0 +1,91 @@
+# ESLint
+
+Author: [Nicholas Chua](https://github.com/nus-cs3281/2018/blob/master/students/nicholasChua/nicholasChua-Resume.md)
+
+## Overview
+
+[ESLint](https://eslint.org/docs/about/) is an open source JavaScript linter.
+
+Code linting is a type of static analysis that helps find problematic patterns or code that does not follow a certain style guideline.
+
+Linting is especially important when working with JavaScript. JavaScript, being a loosely-typed, interpreted language, is prone to developer errors. ESLint identifies errors for developers before it is executed.
+
+## Features
+
+### In Built Rules
+[In built rules](https://eslint.org/docs/rules/) in ESLint are split into a few categories.
+
+* Possible Errors
+* Best Practices
+* Variables
+* Strict Mode
+* Node.js and CommonJS
+* Stylistic Issues
+* ECMAScript 6 (ES6)
+
+### Custom Rules
+Custom rules can be imported via Plugins. You can [learn how to write your own](https://eslint.org/docs/developer-guide/working-with-plugins) or [use existing ones](https://www.npmjs.com/browse/keyword/eslintplugin) that are already published to npm.
+

```

Comment:
How about giving an example of a `custom rule` which can be added using a plugin